rst_files/getting_started.rst

```diff
@@ -63,311 +52,52 @@ install the latest version `Anaconda <https://www.anaconda.com/download/#macos>`
     * We do not recommend `JuliaPro <https://juliacomputing.com/products/juliapro.html>`_
       due to its limited number of available packages
 
-* Open Julia, by navigating to Julia through your menus or desktop icons
-
-Either way you should now be looking at something like this (modulo your operating system)
-
-.. figure:: /_static/figures/julia_term_1.png
-   :scale: 100%
-
-This is called the JULIA *REPL* (Read-Evaluate-Print-Loop), which we discuss more :ref:`below <julia_repl>`
-
-* In the Julia terminal, type the following
-
-    .. code-block:: julia
-        :class: no-execute
-
-        ] add IJulia InstantiateFromURL; precompile
-
-This installs enough Julia packages to begin using the lecture notes
-
-*Note:* On OS/X you will need to type the ``]`` separately and cannot copy/paste the whole string
-
-* To run Jupyter, open a terminal or windows console, ``cd`` to the location you wish to write files and type
-
-.. code-block:: none
-
-    jupyter lab
-
-
-.. _jl_jupyter:
-
-Using Jupyter
-================
-
-
-.. _ipython_notebook:
-
-Getting Started
------------------------
-
-After you have started Jupyter, your web browser should open to a page on the
-local machine that looks something like this
-
-.. figure:: /_static/figures/starting_nb_julia.png
-   :scale: 100%
-
-The page you are looking at is called the "dashboard"
-
-If you click on "Julia 1.0.x" you should have the option to start a Julia notebook
-
-Here's what your Julia notebook should look like
-
-.. figure:: /_static/figures/nb2_julia.png
-   :scale: 100%
-
-The notebook displays an *active cell*, which you can type Julia commands into
-
-.. Not sure this is helpful
-.. **Note** The address ``localhost:8888/lab`` you see in the image indicates that the browser is communicating with a Jupyter lab session via port 8888 of the local machine
-
+.. _intro_repl:
 
-Using QuantEcon Lecture Packages
--------------------------------------------
+* Open Julia, by either:
 
-To use the curated set of packages in the QuantEcon lecture notes,
-put the following text in a notebook cell, and hit ``Shift-Enter`` to run the cell
+    #. Navigating to Julia through your menus or desktop icons (Windows, OSX), or
 
-    .. literalinclude:: /_static/includes/deps_no_using.jl
+    #. Opening a terminal and typing ``julia`` (Linux)
```

Comment:
This also applies to OS X right? Could we also just say Mac maybe?

Reply:
It can work on OSX, but you need to do a `ln -s path_to_julia /usr/local/bin/julia` or whatever.

Reply:
I'm not sure RE calling from the terminal. From the Julia platform-specific instructions: 

> Installation is the same as any other Mac software -- copy the Julia-<version>.app to your hard-drive (anywhere) or run from the disk image. 

So there isn't really an "installer" exactly, as much as a disk image (`.dmg`) that provides the binary. 

And you're right about the space.